### ISSUE TITLE
Strange behavior when using hBox with unicode char

### ISSUE THREAD

**Issue by sifmelcara** [jtdaugherty/brick#61]
``` Haskell
module Main where

import qualified Brick.Main as M
import qualified Brick.Widgets.Core as WC

ui = WC.vLimit h . WC.hLimit 5 $ WC.hBox
    [ WC.str text2 
    , WC.vLimit h $ WC.fill '$'
    ]
    where
    h = 3
    text1 = "    \n寬字\n    \n"
    text2 = "    \nabcd\n    \n"

main :: IO ()
main = M.simpleMain ui
```

The program displays

```
    $
abcd$
    $
```

But when I replace `WC.str text2` by `WC.str text1`, it displays

```

寬字 

```

Both `abcd` and `寬字` occupy 4 display length in my terminal.
Maybe there is something wrong with wide character rendering?

**Comment by jtdaugherty** [comment]
Right now, wide characters are not supported.  I've had it on my TODO list for some time, but I haven't spent any time investigating what would be required.  (I supported this in `vty-ui` previously so I'm familiar with the problem and its solution; I just haven't dedicated time to fixing this in `brick`.)

I'll take a look at this in the next few days.

Thanks!

**Comment by jtdaugherty** [comment]
I started to implement this and ran into a problem with Vty, so this will have to wait until Vty moves forward a bit (see https://github.com/coreyoconnor/vty/issues/94).

**Comment by jtdaugherty** [comment]
This is now fixed and is released on Hackage in version `0.14`.
